public/assets/map/gameObjects.js

```diff
@@ -9,16 +9,16 @@ const getStartingTileTree = getStartingTile;
 
 module.exports = [
   {
-    name: "house1",
-    displayName: "house",
+    name: "house_dire",
```

Comment:
Please make something like
`const FRACTIONS = [
  {
    name: "Radiant",
    houses: [{ name: "house_radiant", displayName: "Radiant Sanctuary"}]
  },
];`
and use it here and in `server/mocks/players.js`